src/app/qgsoptions.cpp

```diff
@@ -270,6 +270,21 @@ QgsOptions::QgsOptions( QWidget *parent, Qt::WindowFlags fl )
     mListHiddenBrowserPaths->addItem( newItem );
   }
 
+  //locations of the QGIS help
+  myPaths = mSettings->value( QStringLiteral( "help/helpSearchPath" ), "" ).toString();
```

Comment:
This is fragile (I realise it's copied/pasted from elsewhere, but easier to fix new settings then existing ones). A better approach:

    QStringList helpPathList = mSettings->value( QStringLiteral( "help/helpSearchPath" ) ).toStringList();
    Q_FOREACH( const QString& path, helpPathList )
    {
        QTreeWidgetItem* item = new QTreeWidgetItem();
        item->setText( 0, path );
        item->setFlags( Qt::ItemIsEnabled | Qt::ItemIsSelectable | Qt::ItemIsEditable );
        mHelpPathTreeWidget->addTopLevelItem( item );
    }

This also has the advantage of
1. avoiding the ugly "my" prefixed variables
2. no issues if someone has a | in their paths
3. no use of "" literal (`QString()` should be used instead)